src/app/script/dialog_class.cpp

```diff
@@ -939,6 +939,28 @@ int Dialog_modify(lua_State* L)
       }
     }
     lua_pop(L, 1);
+    // Handling options before option should support
+    // using both or only one of them at the same time
+    type = lua_getfield(L, 2, "options");
+    if (type != LUA_TNIL) {
+      if (lua_istable(L, -1)) {
+        if (auto combobox = dynamic_cast<ui::ComboBox*>(widget)) {
+          combobox->deleteAllItems();
+        lua_pushnil(L);
+        bool empty = true;
+        while (lua_next(L, -2) != 0) {
+          if (auto p = lua_tostring(L, -1)){
+            combobox->addItem(p);
+            empty = false;
+          }
+          lua_pop(L, 1);
+        }
+        if (empty)
+          combobox->getEntryWidget()->setText("");
```

Comment:
It looks like we need an extra indentation level here:
```suggestion
          lua_pushnil(L);
          bool empty = true;
          while (lua_next(L, -2) != 0) {
            if (auto p = lua_tostring(L, -1)){
              combobox->addItem(p);
              empty = false;
            }
            lua_pop(L, 1);
          }
          if (empty)
            combobox->getEntryWidget()->setText("");
```